numpy/core/tests/test_longdouble.py

```diff
@@ -97,6 +98,20 @@ def test_fromfile_bogus(self):
             res = np.fromfile(path, dtype=float, sep=" ")
         assert_equal(res, np.array([1., 2., 3.]))
 
+    def test_fromfile_stdout(self):
+        p = sp.Popen(['echo numpy is not np'],
+                      stdout=sp.PIPE, shell=True)
+        arr = np.fromfile(p.stdout, dtype=np.uint8, count=len('numpy is not np\n'))
+        assert_array_equal(arr.tostring(),
+                           np.asarray('numpy is not np\n', dtype='|S'))
+
+    def test_fromfile_stdout_unbuffered(self):
+        p = sp.Popen(['echo numpy is not np'],
```

Comment:
I'd perhaps be tempted to run `[sys.executable, 'print("numpy is not np")']` instead, to avoid hitting any shell weirdness

Reply:
I keep getting:
```
In [5]: a = sp.Popen([sys.executable, 'print("numpy is not np")'], stdout=sp.PIPE)                                                                            

In [6]: /home/mark2/miniconda3/envs/npdev/bin/python: can't open file 'print("numpy is not np")': [Errno 2] No such file or directory
```

Reply:
Got it to work eventually. thanks!